src/app/icd/IcdMonitoringTable.cpp

```diff
@@ -5,32 +5,51 @@ namespace chip {
 
 enum class Fields : uint8_t
 {
-    kCheckInNodeID    = 1,
-    kMonitoredSubject = 2,
-    kKey              = 3,
+    kCheckInNodeID = static_cast<uint8_t>(PersistentTags::kEntries),
```

Comment:
Wait, why are we using the same tag value as someone else?

And do these tag values actually need to be distinct from the ones for PersistentArray?